src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -119,6 +119,11 @@ function isSubmitAction(
         return false;
     }
 
+    const hasOnlyPendingTransactions = reportTransactions.length > 0 && reportTransactions.every((transaction) => isPending(transaction));
+    if (hasOnlyPendingTransactions) {
```

Comment:
I think better approach is to see if there is at least one non-pending transaction so we exit early instead of always iterating over all the transactions